packages/application-shell/src/components/application-page-title/application-page-title.tsx

```diff
@@ -0,0 +1,127 @@
+import type { TEnhancedLocation } from '@commercetools-frontend/browser-history';
+
+import { useLayoutEffect } from 'react';
+import { useLocation } from 'react-router-dom';
+import startCase from 'lodash/startCase';
+import capitalize from 'lodash/capitalize';
+
+type Breadcrumb<Query extends {} = {}> = {
+  suffix: string;
+  paths: string[];
+  location?: TEnhancedLocation<Query>;
+};
+type Props = {
+  /**
+   * Allow to render a custom page title.
+   * Overrides default page title with the format:
+   * <?resource_name(manually inputed)> - <?page_location> - <application_identifier> - <project_key> - Merchant Center
```

Comment:
I find a little confusing this explanation.
Perhaps we could go for something like this:
```
Allows to extend the page title. 
When provided, the array items will be used as a prefix for the default title this component will generate.

It will follow a pattern like this:
<renderPageTitle.join(' - ')> - <?page_location> - <application_identifier> - <project_key> - Merchant Center
```

What do you think?

Reply:
Oh yes, this makes sense too. I just wanted to provide some clarity to how the titles would be formed from an additional content provided.

Reply:
updated here: https://github.com/commercetools/merchant-center-application-kit/pull/2758/commits/07686cec816448de631b4144c58f8c5cd3eddc28